recipes/gperf/all/conanfile.py

```diff
@@ -23,13 +25,13 @@ def _is_mingw_windows(self):
 
     def build_requirements(self):
         if self.settings.os == "Windows" and tools.os_info.is_windows:
-            if "CONAN_BASH_PATH" not in os.environ and tools.os_info.detect_windows_subsystem() != 'msys2':
+            if "CONAN_BASH_PATH" not in os.environ and tools.os_info.detect_windows_subsystem() != "msys2":
                 self.build_requires("msys2/20190524")
```

Comment:
I believe is should be like this now

```suggestion
                self.build_requires("msys2/cci.latest")
```

Reply:
No problem and thanks! Committed with the patch.